cmd/entrypoint/cmd/agent.go

```diff
@@ -30,7 +31,11 @@ const (
 type logger struct{}
 
 func (l logger) Println(v ...interface{}) {
-	log.Error(fmt.Sprint(v...), nil)
+	log.Error("[promhttp] "+fmt.Sprint(v...), nil)
+}
+
+func (l logger) Print(v ...interface{}) {
+	log.Error("[mysql] "+fmt.Sprint(v...), nil)
 }
```

Comment:
The source of the log (promhttp or mysql) is distinguished by the called function. This is not good design. We have to provide different logger type.

Reply:
Thank you. I make two types of loggers.